packages/orm/src/getProperties.ts

```diff
@@ -33,7 +33,9 @@ export function getProperties(meta: EntityMetadata): Record<string, any> {
   // on fake instances" getters to throw nonsense errors anyway (which we suppress), so it should be fine.
   const cached = (propertiesCache[key] = {});
 
-  const instance = getFakeInstance(meta);
+  const fakeEm = undefined as any;
+  const instance = new (meta.cstr as any)(fakeEm, true);
```

Comment:
This is the only place we call `new` to invoke the constructors & field initializers, so we can capture the ones that are `RelationConstructor`s and later turn them into getters.